files/lib/data/conversation/ConversationAction.class.php

```diff
@@ -310,6 +315,8 @@ public function validateMarkAsRead() {
 				$this->parameters['visitTime'] = TIME_NOW;
 			}
 		}
+
+		unset($this->parameters['userID']);
```

Comment:
Sure? Because it will be set (https://github.com/WoltLab/com.woltlab.wcf.conversation/pull/103/files#diff-7a7e3cc9859dcffe5f602f153fea79f8R221) if the array key is not set and the `finalizeAction` event will be called after the `markAsRead` method, so it should be set in anyw ay?